setup.py

```diff
@@ -19,9 +19,9 @@
 # You should have received a copy of the GNU General Public License
 # along with git-big-picture.  If not, see <http://www.gnu.org/licenses/>.
 
-from setuptools import setup
+from setuptools import find_packages, setup
 
-from git_big_picture import __version__ as VERSION
+from git_big_picture._main import __version__ as VERSION
```

Comment:
Can you maybe `from ._main import __version__` in the `__init__.py` instead to circumvent this. It would be nice if the `__version__` is exported at the top level so you can do `from git_big_picture import __version__` and such.

Reply:
I don't mind, sure.